jnigen/lib/src/bindings/dart_generator.dart

```diff
@@ -502,11 +549,11 @@ class _TypeClass {
 
 /// Generates the type class.
 class _TypeClassGenerator extends TypeVisitor<_TypeClass> {
+  _TypeClassGenerator(this.resolver, {this.isConst = true});
```

Comment:
nit: I think it's more common to have fields first, then constructors/factories, then static methods, and then normal methods.

But, I might be projecting from the C++ code style. (I don't see any mention of this in Dart style.)